uint8 ACTION_READ=0
uint8 ACTION_WRITE=1
uint8 dev_address
uint8 register_address
uint8 action
uint8[] data
uint8 read_size
---
uint8[] response
